plugins/wacom-usb/fu-wac-module-sub-cpu.c

```diff
@@ -0,0 +1,251 @@
+/*
+ * Copyright 2024 Jason Gerecke <jason.gerecke@wacom.com>
+ *
+ * SPDX-License-Identifier: LGPL-2.1-or-later
+ */
+
+#include "config.h"
+
+#include <string.h>
+
+#include "fu-wac-common.h"
+#include "fu-wac-device.h"
+#include "fu-wac-module-sub-cpu.h"
+#include "fu-wac-struct.h"
+
+struct _FuWacModuleSubCpu {
+	FuWacModule parent_instance;
+};
+
+G_DEFINE_TYPE(FuWacModuleSubCpu, fu_wac_module_sub_cpu, FU_TYPE_WAC_MODULE)
+
+#define FU_WAC_MODULE_SUB_CPU_PAYLOAD_SZ      256
+#define FU_WAC_MODULE_SUB_CPU_START_NORMAL    0x00
+
+static FuChunk *
+fu_wac_module_sub_cpu_create_chunk(GPtrArray *srec_records, guint32 *record_num, GError **error)
+{
+	g_autoptr(GByteArray) data = g_byte_array_new();
+	g_autoptr(GBytes) blob;
+	FuChunk *chunk;
+	guint32 base_addr = 0;
+	guint32 expect_addr = 0;
+
+	for (; *record_num < srec_records->len; *record_num += 1) {
+		FuSrecFirmwareRecord *rcd = g_ptr_array_index(srec_records, *record_num);
+		GByteArray *src = rcd->buf;
+
+		/* Skip non-data records */
```

Comment:
single line comments don't have a sentence case --there's a few of these here -- sorry for the nits!

Reply:
Ack. (also: I totally understand the shower of nits --- I've had to hand out my share in code reviews too :smile: it's hard to keep the various project preferences straight sometimes)